irispy/spectrograph.py

```diff
@@ -73,11 +73,11 @@ def __repr__(self):
         obs_period, instance_period = result
         return ("<iris.IRISSpectrograph instance\nOBS ID: {obsid}\n"
                "OBS Description: {obsdesc}\n"
-               "OBS period: {obs_period}\n"
+               "OBS Period: {obs_period}\n"
                "Instance period: {inst_period}\n"
-               "Number unique raster positions: {nraster}\n"
-               "Number of repeats: {repeats}\n"
-               "Spectral windows : dimensions [raster axis, slit axis, spectral axis]:"
+               "OBS Number unique raster positions: {nraster}\n"
+               "OBS Number of repeats: {repeats}\n"
```

Comment:
This value is now actually being taken from the data, rather than metadata describing the OBS as a whole.  So I think it should be ```"Instance Number of repeats"```.  In fact I think it's better to remove it altogether as the number of repeats in each instance is given by dimensions below.